src/components/TreesMap/TreesMap.tsx

```diff
@@ -603,7 +608,11 @@ export const TreesMap = forwardRef<MapRef, TreesMapPropsType>(function TreesMap(
               ? {
                   '': (
                     <StyledTextLink
-                      href={getOSMEditorURL(pumpInfo.id)}
+                      href={getOSMEditorURL({
+                        nodeId: pumpInfo.id,
+                        lat: pumpInfo.lat,
+                        lon: pumpInfo.lon,
```

Comment:
I don't think the lat/lon are helpful here, since the nodeId will make MapComplete first pull the data and then center the map on this geometry. It did work before, so no real need to add it, right?

Reply:
@tordans During my testing MapComplete did not center nor zoom into the pump the user clicked on gdk. That is why I added this.

Reply:
I consider this a Bug in MapComplete and will report it there. It did behave like this before the big refactoring recently.

Reply:
@ff6347 for me, the auto-centering works better than the added lat/lng you are adding now. The difference is, that the auto-centering takes the open action panel into account.

- with auto center https://mapcomplete.org/theme.html?userlayout=https%3A%2F%2Fraw.githubusercontent.com%2Ftechnologiestiftung%2FMapComplete-ThemeHelper%2F024ef9134987e0ead34f261a6270d5572c4d31a4%2FOSM-Berlin-Themes%2Fman_made-walter_well-status-checker%2Ftheme.json&language=de#node/1456288912
- your new link https://mapcomplete.org/?userlayout=https%3A%2F%2Fraw.githubusercontent.com%2Ftechnologiestiftung%2FMapComplete-ThemeHelper%2F024ef9134987e0ead34f261a6270d5572c4d31a4%2FOSM-Berlin-Themes%2Fman_made-walter_well-status-checker%2Ftheme.json&z=18&lat=52.4715839&lon=13.46742&language=de#node/1456288912 with manual lat/lng

Not a huge difference, but personally I would try to keep the part of the code here simple and let Mapcomplete handle the details…

Reply:
Hey @tordans So was there a bug that got fixed? I swear it did not center nor set the zoom before.

Reply:
I did not read about it on the MapComplete Repo, but it might be…
Or maybe the lookup for the specific note you tested was broken…
Who knows :)